services/api/src/rsu_querycounts.py

```diff
@@ -5,15 +5,28 @@
 import logging
 from pymongo import MongoClient
 
+message_types = {
```

Comment:
same comment as above here (psm addition or pull out to an env variable).

Reply:
Added PSM